src/main/java/com/redhat/cloud/policies/app/lightweight/AccountLatestUpdateRepository.java

```diff
@@ -23,4 +23,13 @@ public void setLatestToNow(String accountId) {
                 .setParameter("latest", LocalDateTime.now(UTC))
                 .executeUpdate();
     }
+
+    public void setLatestToNowOrgId(String orgId) {
+        String sql = "INSERT INTO account_latest_update (account_id, latest) SELECT :orgId, :latest " +
+                "ON CONFLICT (org_id) DO UPDATE SET latest = :latest";
```

Comment:
one way is using the correct org_id instead of account_id.
Another way is adding both account_id and org_id in case the org_id is there. 

I don't see an elegant way of how to aggregate these two methods without having something unmaintainable. Maybe you have something in mind? Otherwise I would stick with it and do the easiest thing that works as we will delete one of these methods anyway.

Reply:
For this specific migration, I would probably duplicate the `account_latest_update` table into an `org_latest_update` table and query it when the `orgId` feature is enabled. The `account_latest_update` would be dropped when the migration is over.

Reply:
ok. Sounds ok as well. Added :)